INSTALL.md

```diff
@@ -319,9 +330,9 @@ python3 examples/core_classic_occ_bottle.py
 ```bash
 >>> ImportError: DLL load failed while importing _gp:
 ```
-set the ```OCCT_ESSENTIALS_PATH``` environment variable
+set the ```OCCT_ESSENTIALS_ROOT``` environment variable
 ```batch
-setx OCCT_ESSENTIALS_PATH "%CASROOT%\win64\vc14\bin"
+setx OCCT_ESSENTIALS_ROOT "%CASROOT%\win64\vc14\bin"
```

Comment:
The "OCCT_ESSENTIALS_ROOT" must be the root of OpenCASCADE in which there are not only opencascade itself, but also its 3rd party libraries. Otherwise, many packages (AIS etc.) would throw DLL not found error.
The recommended folder structure is described in OCCT's documentation here  https://dev.opencascade.org/doc/overview/html/index.html#intro_install_windows